src/main/java/ConvertFormatJsonToXml.java

```diff
@@ -0,0 +1,15 @@
+import exceptions.ConvertFormatXmlToJsonException;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.XML;
+
+
+public class ConvertFormatJsonToXml {
```

Comment:
I would suggest that you create an abstract Formats Handler class.
and then you will have JsonFormatReader class and XMLFormarWriter Class, and comine them together.

Tommorow you will want to convert Json to avro or Xml to Json, and it will be very difficult in this way.